subprojects/packagefiles/emilk-loguru/meson_options.txt

```diff
@@ -104,8 +104,7 @@ option(
 
 option(
   'loguru_stacktraces',
-  type: 'boolean',
-  value: true,
+  type: 'feature',
```

Comment:
We could add `meson_version: '>=0.47.0'` to the project() declaration too, I guess.

Reply:
feature is already used elsewhere. but sure.

Reply:
Ah, in that case it *definitely* can't hurt. :D